### PR TITLE
Modify the registry address and namespace of the ADDONS

### PR DESCRIPTION
**Pull request by xiaoluhong**

### CODE REVIEW

File: package/addons-update.sh [hunk 1]
```diff
@@ -51,13 +51,13 @@ metadata:
     addonmanager.kubernetes.io/mode: EnsureExists
 EOF
 
-GCR_IO_REGISTRY=${REGISTRY:-gcr.io}
+GCR_IO_REGISTRY=${REGISTRY:-docker.io}
```
Comment (alena1108) [inline package/addons-update.sh:54]: @hongxiaolu lets rename the parameter to align with the default registry name. Renaming would require changing the references in addon manifests.

Comment (deniseschannon) [reply package/addons-update.sh:54]: @hongxiaolu it looks like the docker.io param already exists https://github.com/rancher/kubernetes-package/blob/k8s-v1.9/package/addons-update.sh#L55

so it just needs to be replaced in the add-ons yamls as well as in the catalog PR.

Comment (deniseschannon) [reply package/addons-update.sh:54]: we should also remove any reference to the GCR registry to ensure we didn't miss anything.

**Comment by deniseschannon** [comment]
closing in favor of https://github.com/rancher/kubernetes-package/pull/133

**Pull request closed without merging**